diff --git a/bindings/cs/bgfx.cs b/bindings/cs/bgfx.cs
index b3e368303..fbed760ce 100644
--- a/bindings/cs/bgfx.cs
+++ b/bindings/cs/bgfx.cs
@@ -12,9 +12,9 @@
 using System;
 using System.Runtime.InteropServices;
 
-namespace Bgfx
+namespace BgfxCS
 {
-public static partial class bgfx
+public static partial class Bgfx
 {
 	[Flags]
 	public enum StateFlags : ulong
diff --git a/bindings/cs/bgfx_dllname.cs b/bindings/cs/bgfx_dllname.cs
index 69aaacb5d..0057e822b 100644
--- a/bindings/cs/bgfx_dllname.cs
+++ b/bindings/cs/bgfx_dllname.cs
@@ -13,14 +13,10 @@
  *
  */
 
-namespace Bgfx
+namespace BgfxCS
 {
-public static partial class bgfx
+public static partial class Bgfx
 {
-#if DEBUG
-       const string DllName = "bgfx_debug.dll";
-#else
-       const string DllName = "bgfx.dll";
-#endif
+    const string DllName = "bgfx.dll";
 }
 }
